### PR TITLE
Rename Avro ScioContext ops class

### PR DESCRIPTION
**Pull request by regadas**

### CODE REVIEW

File: scio-avro/src/main/scala/com/spotify/scio/avro/syntax/ScioContextSyntax.scala [hunk 1]
```diff
@@ -31,7 +31,7 @@ import scala.reflect.ClassTag
 import scala.reflect.runtime.universe._
 
 /** Enhanced version of [[ScioContext]] with Avro methods. */
-final class AvroScioContext(private val self: ScioContext) extends AnyVal {
+final class ScioContextOps(private val self: ScioContext) extends AnyVal {
```
Comment (andrisnoko) [inline scio-avro/src/main/scala/com/spotify/scio/avro/syntax/ScioContextSyntax.scala]: @regadas why rename? `AvroScioContext` seems to be appropriately named as in others like `BigQueryScioContext`

Comment (regadas) [reply scio-avro/src/main/scala/com/spotify/scio/avro/syntax/ScioContextSyntax.scala]: these are `Ops` that are added to `ScioContext` and the package is `com.spotify.scio.avro.syntax`. Avro prefix is a bit redundant.

Comment (regadas) [reply scio-avro/src/main/scala/com/spotify/scio/avro/syntax/ScioContextSyntax.scala]: #1763 is another example

Comment (regadas) [reply scio-avro/src/main/scala/com/spotify/scio/avro/syntax/ScioContextSyntax.scala]: in regards to `BigQueryScioContext` will be proposing the same thing.

File: scio-avro/src/main/scala/com/spotify/scio/avro/syntax/ScioContextSyntax.scala [hunk 2]
```diff
@@ -80,5 +80,5 @@ final class AvroScioContext(private val self: ScioContext) extends AnyVal {
 
 /** Enhanced with Avro methods. */
 trait ScioContextSyntax {
-  implicit def toAvroScioContext(c: ScioContext): AvroScioContext = new AvroScioContext(c)
+  implicit def toScioContextOps(c: ScioContext): ScioContextOps = new ScioContextOps(c)
```
Comment (jto) [inline scio-avro/src/main/scala/com/spotify/scio/avro/syntax/ScioContextSyntax.scala]: Would that not break the implicit conversion if `BigQueryScioContext` and `AvroScioContext` have the implicit conversion with the same name that are  both in scope ?

Comment (regadas) [reply scio-avro/src/main/scala/com/spotify/scio/avro/syntax/ScioContextSyntax.scala]: @jto 🤦‍♂️ yes! 💣

Comment (regadas) [reply scio-avro/src/main/scala/com/spotify/scio/avro/syntax/ScioContextSyntax.scala]: @jto I'll try to go with this convention `def <main package><ops class>( ... )`

Comment (jto) [reply scio-avro/src/main/scala/com/spotify/scio/avro/syntax/ScioContextSyntax.scala]: sounds good

**Pull request merged**